generators/app/templates/javascript/storage/_package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.1",
+  "description": "<%= description %>",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": ["lib/"],
+  "engines": {
+    "node": ">=10"
+  },
+  "dependencies": {
+    "@verdaccio/commons-api": "^8.4.2",
+    "express": "^4.17.1"
```

Comment:
Why `express` in `storage` plugin ? is it needed ?

Reply:
My bad, i just copied the package Json, I will clean it up in the next PR

Reply:
cool 👍